static/app/views/teamInsights/teamAlertsTriggered.tsx

```diff
@@ -72,31 +72,38 @@ class TeamIssues extends AsyncComponent<Props, State> {
 
   renderBody() {
     const {alertsTriggered} = this.state;
+    const data = Object.entries(alertsTriggered ?? {})
```

Comment:
The use of `data` identifier is not clear, can we elaborate on the identifier e.g `alertData`